### ISSUE TITLE
Unused throws should consider methods declared in methodThrowTypeDeclarations

### ISSUE THREAD

**Issue by marcospassos** [pepakriz/phpstan-exception-rules#57]
The unused throws rule should consider the methods declared as well:

```php
class Foo {
   /**
    * @throws FooException
    */
   public function bar() : void { // Unused @throws FooException annotation
       \call_user_func($this->callable);    
   } 
}
```

```neon
exceptionRules:
     methodThrowTypeDeclarations:
        Foo:
            bar: [FooException]
```

**Comment by marcospassos** [comment]
I've already fixed in [this branch](https://github.com/marcospassos/phpstan-exception-rules/compare/master...marcospassos:fix-consider-default-as-used?expand=1), just waiting for #54 be merged to open the PR.

**Comment by pepakriz** [comment]
Let's open the PR :)

**Comment by marcospassos** [comment]
Fixed in #59
